src/App.js

```diff
@@ -3,13 +3,107 @@ import Form from "./Form";
 import "./App.css";
 
 class App extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      userInput: [],
+      operation: "",
+      result: "",
+    }
+  }
+
+  handleClick = (event) => {
+    const { operation, userInput } = this.state
+    event.preventDefault();
+    switch (operation) {
+      case "sum":
+        this.handleSumChange(userInput);
+        break;
+      case "average":
+        this.handleAvgChange(userInput);
+        break;
+      case "mode":
+        this.handleModeChange(userInput);
+        break;
+      default:
+        this.handleDefault();
+    }
+  }
+
+  handleUserInput = (event) => {
+    const numsInput = event.target.value.split(",").map((el) => {
+      return el;
+    })
```

Comment:
```suggestion
    const numsInput = event.target.value.split(",")
```
.map isn't doing anything here, so you don't need it!